nicegui/native/native_mode.py

```diff
@@ -21,14 +21,32 @@
         # webview depends on bottle which uses the deprecated CGI function (https://github.com/bottlepy/bottle/issues/1403)
         warnings.filterwarnings('ignore', category=DeprecationWarning)
         import webview
+        from webview.dom import DOMEventHandler
     optional_features.register('webview')
 except ModuleNotFoundError:
     pass
 
 
+def on_drop(e: dict[str, Any], drop_queue: mp.Queue):
+    files = e['dataTransfer']['files']
+    if len(files) == 0:
+        return
+
+    for file in files:
+        drop_queue.put(file.get('pywebviewFullPath'))
+
+
+def bind(window: webview.Window, drop_queue: mp.Queue) -> None:
+    window.dom.document.events.drop += DOMEventHandler(
+        lambda e: on_drop(e, drop_queue),
+        True,
+        True,
+    )
```

Comment:
```suggestion
    window.dom.document.events.drop += DOMEventHandler(lambda e: on_drop(e, drop_queue), True, True)
```

No need for 5 lines.

Oh and mypy is complaining:
`Argument 1 to "__iadd__" of "Event" has incompatible type "DOMEventHandler"; expected "Callable[..., Any]"Mypy[arg-type](https://mypy.readthedocs.io/en/latest/_refs.html#code-arg-type)`

Reply:
I don't really know so I asked Claude and got this answer:
````
The window.dom.document.events.drop is an event from pywebview that expects to have handlers added to it, but mypy sees it as a standard Python Event (like from the threading module) which has a different expectation for its __iadd__ method.
```

I added `# type: ignore[arg-type]` after the string but it becomes too long for autopep8 so it places the comment on a new line.
I saved the file without formatting but for it to save with auto formatting I think we'll have to split the line up into multiple lines to let `# type: ignore[arg-type]` be on the correct line.
Should I do that?